design/apidsl/action.go

```diff
@@ -274,6 +274,20 @@ func Params(dsl func()) {
 //	})
 //
 func Payload(p interface{}, dsls ...func()) {
+	payload(false, p, dsls...)
+}
+
+// OptionalPayload implements the action optional payload DSL. The function works identically to the
+// Payload DSL except it would set a bit in the action definition to denote that the payload is not
```

Comment:
replace `except it would set` with `except it sets`
replace `Examples:` with `Example:`
replace 
```
// Request payload is described by the BottlePayload type
```
with
```
// Request payload is described by the BottlePayload type and is optional
```